tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousTest.java

```diff
@@ -65,34 +68,20 @@ public static WebArchive deploy() {
      */
     @Test
     public void testAsyncIsNotFinished() {
-        Future<Connection> future = null;
-        try {
-            future = client.service();
-        }
-        catch (InterruptedException e) {
-            throw new AssertionError("testAsync: unexpected InterruptedException calling service");
-        }
-
+        CompletableFuture<Void> waitingFuture = newWaitingFuture();
+        Future<Connection> future = client.service(waitingFuture);
```

Comment:
Can we ensure that the waitingFuture is completed, otherwise the asynchronous task may run forever?

Reply:
We can use the same pattern we use in other tests to ensure that waitingFutures are closed at the end of the test, I'll find an example.

Reply:
See these parts of AsynchronousCSTest:
* https://github.com/eclipse/microprofile-fault-tolerance/blob/master/tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousCSTest.java#L61
* https://github.com/eclipse/microprofile-fault-tolerance/blob/master/tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousCSTest.java#L250
* https://github.com/eclipse/microprofile-fault-tolerance/blob/master/tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/AsynchronousCSTest.java#L264

Reply:
done